modules/acquia_cms_headless/acquia_cms_headless.module

```diff
@@ -4,3 +4,21 @@
  * @file
  * Contains hook implementations for the acquia_cms_headless module.
  */
+
+use Drupal\user\RoleInterface;
+
+/**
+ * Implements hook_content_model_role_presave_alter().
+ */
+function acquia_cms_headless_content_model_role_presave_alter(RoleInterface &$role) {
+  $role_id = $role->id();
+  switch ($role->id()) {
+    case 'content_administrator':
+    case 'content_author':
+    case 'content_editor':
+      $user = user_load_by_name('Headless');
```

Comment:
@vishalkhode1 I don't think changing the machine name of the role is acceptable, if someone does that which means people breaking functionality.

Reply:
> @vishalkhode1 I don't think changing the machine name of the role is acceptable, if someone does that which means people breaking functionality.

It's not about the machine name, it's about the label. The label can be changed. Right ?

Reply:
What I see from the PR is the @rajeshreeputra is using role machine name to load user.

Reply:
@panshulK We are loading user here by username. What I meant is that: Username can be changed (Though people do it very rare, but since we are storing uuid of the user in the config, then let's use that).
@rajeshreeputra

Reply:
Understood @vishalkhode1 . Thanks